### PR TITLE
Fix issue

### PR DESCRIPTION
**Pull request by obadrawi**
-UI: Feedback for event sending #206
-Differentiate API type by version #209
-update dependencies

### CODE REVIEW

File: modules/api-server/src/server/services.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ function fillEventData(event: any) {
         specInJson = yaml.safeLoad(fs.readFileSync(event.specification, 'utf8'))
     }
     let labels = event.labels ? event.labels : {};
-    labels["type"] = "AsyncApi";
+    labels["type"] = "AsyncApi v" + specInJson.asyncapi;
```
[the remark this answers is not in the document]
Comment (a-thaler) [reply modules/api-server/src/server/services.ts]: I think we still should use the major version only as the major version is the interesting one from compatinility perspective

File: modules/api-server/src/server/services.ts [hunk 2]
```diff
@@ -188,6 +188,18 @@ function fillServiceMetadata(api: any, baseUrl: any) {
     }
     let labels = api.labels ? api.labels : {};
     labels["type"] = api.type == "odata" ? "OData" : "OpenAPI"
+    if (api.type == "odata") {
+        labels["type"] = "OData v" + 2
+    }
+    else if (apiData.spec.openapi) {
+        labels["type"] = "OpenAPI v" + apiData.spec.openapi
```
Comment (a-thaler) [inline modules/api-server/src/server/services.ts]: I would really prefer to use only the major version, can you add a simple parsing logic?

File: modules/cockpit/src/app/sendEventView/send.eventview.ts [hunk 3]
```diff
@@ -73,6 +74,8 @@ export class SendEventViewComponent implements OnInit {
                 .subscribe(
                     success => {
                         this.loading = false;
+                        this.successMessage = "Event has been sent Successfully";
```
Comment (a-thaler) [inline modules/cockpit/src/app/sendEventView/send.eventview.ts]: Please ask Barbara for language review of that sentence.

File: modules/api-server/src/server/services.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (obadrawi) [reply modules/api-server/src/server/services.ts]: done

**Pull request merged**